app/src/main/java/org/dosomething/letsdothis/ui/RegisterActivity.java

```diff
@@ -256,6 +256,7 @@ public void onEventMainThread(RegisterTask task)
     {
         if(AppPrefs.getInstance(this).isLoggedIn())
         {
+            broadcastLogInSuccess(this);
             Toast.makeText(this, "success register", Toast.LENGTH_SHORT).show();
             startActivity(MainActivity.getLaunchIntent(this));
             finish();
```

Comment:
I don't think we'll need this anymore since it should be handled by the broadcast receiver. I'll test.